examples/apps/zoom-app/requirements.txt

```diff
@@ -0,0 +1,2 @@
+landingai
```

Comment:
Can we fix the current version here? So that end users copying this example will not be affected by future breaking changes.